### ISSUE TITLE
add longer relative interval time ranges 

### ISSUE THREAD

**Issue by bernhardreiter** [csaf-poc/csaf_distribution#480]
Allow days (d) for relative time ranges. Avoid the missunderstanding that "m" means "minutes" while user expect months.

With 3.0.0-beta.2 we only have hours (h) and even more fine grained abbrevations, so 
using `-t 2d` will lead to

```
error: invalid argument for flag `-t, --timerange' (expected *models.TimeRange): "2d" is not a valid RFC date time
```

While at it: only document the time ranges at one place in the documentation (currently we have two checker and downloader)

**Comment by bernhardreiter** [comment]
We'd need to add "days", because we assumed it to work for #425.

**Comment by bernhardreiter** [comment]
Some more details: 
@s-l-teichmann informed me, that we had already discussed the technical part: it is hard to define where a day (or a month or a year) as relative interval should start and end. This is (as far as I have understood) one of the reasons why https://pkg.go.dev/time@go1.20.6#ParseDuration does not implement abbreviations for it.

On the other hand the behavior can be surprising, so we made two usability improvements 
 1. #482 with more examples
 2. #481 to see which time limits result from the interpretation

So this is good for now. :)
